code/game/machinery/vending/vendor_types/crew/staff_officer.dm

```diff
@@ -15,6 +15,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
 		list("Headset", 0, /obj/item/device/radio/headset/almayer/mcom, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
 		list("Helmet", 0, /obj/item/clothing/head/helmet/marine/MP/SO, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
 		list("MRE", 0, /obj/item/storage/box/MRE, MARINE_CAN_BUY_MRE, VENDOR_ITEM_MANDATORY),
+		list("Service Jacket", 0, /obj/item/clothing/suit/storage/jacket/marine/service/, MARINE_CAN_BUY_MRE, VENDOR_ITEM_REGULAR),
```

Comment:
The service jacket buy would overlap with MRE buyability. MARINE_CAN_BUY_MRE needs changed to one of the other defines that fits.

Reply:
The service jacket and MRE buyability overlapping were fixed, the service jacket was changed to MARINE_CAN_BUY_ESSENTIALS.